api/tests/routes/backoffice/collective_offers_test.py

```diff
@@ -132,6 +166,7 @@ def test_list_collective_offers_by_name(self, authenticated_client, collective_o
         assert rows[0]["Date de l'évènement"] == (datetime.date.today() + datetime.timedelta(days=3)).strftime(
             "%d/%m/%Y"
         )
+        assert rows[0]["Prix"] == str(collective_offers[1].collectiveStock.price)
```

Comment:
Pour être sûr de tester que c'est bien formaté : 
```suggestion
        assert rows[0]["Prix"] == "11,00 €"
```